src/smt.rs

```diff
@@ -393,6 +393,14 @@ pub fn literal_false() -> SMTExpr {
     literal_bool("false".to_string())
 }
 
+pub fn extract_msb_12_bytes(expr: impl Into<SMTExpr>) -> SMTExpr {
```

Comment:
`extract_ms_12_bytes` (the "b" in "msb" is "bits")
but maybe that way it is not understandable...

Reply:
yea that's also what I thought. `msb` immediately gives you the idea of what's happening, even if the word itself isn't 100% precise